From e4c417c322ec6d40ef373ac0f893d9d95e3f237c Mon Sep 17 00:00:00 2001
From: Danny Nold <dannynold@freescale.com>
Date: Thu, 7 Apr 2011 20:55:18 -0500
Subject: [PATCH] ENGR00140983-2 - FSL Devices: Add LCD device functions to enable/disable pins

- Added function declarations to acquire/release/enable/disable device pins.

Signed-off-by: Danny Nold <dannynold@freescale.com>
---
 include/linux/fsl_devices.h |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/include/linux/fsl_devices.h b/include/linux/fsl_devices.h
index 348884d..c1b5476 100644
--- a/include/linux/fsl_devices.h
+++ b/include/linux/fsl_devices.h
@@ -307,6 +307,10 @@ struct mxc_lcd_platform_data {
 	char *core_reg;
 	char *analog_reg;
 	void (*reset) (void);
+	int (*get_pins) (void);
+	void (*put_pins) (void);
+	void (*enable_pins) (void);
+	void (*disable_pins) (void);
 	int boot_enable;
 };
 
@@ -359,7 +363,7 @@ struct mxc_epdc_fb_mode {
 struct mxc_epdc_fb_platform_data {
 	struct mxc_epdc_fb_mode *epdc_mode;
 	int num_modes;
-	void (*get_pins) (void);
+	int (*get_pins) (void);
 	void (*put_pins) (void);
 	void (*enable_pins) (void);
 	void (*disable_pins) (void);
-- 
1.5.4.4

